pyproject.toml

```diff
@@ -46,7 +46,7 @@ numba = "*"
 Sphinx = "*"
 sphinx-rtd-theme = "*"
 numpydoc = "1.1.0"
-proposal = "6.1.6"
+proposal = ">=7.0.0"
```

Comment:
To keep track which tag/release of NuRadioMC used which proposal version I think we should fix that?

Reply:
I'd say that we should either fix the major version (`proposal = "^7"`) so that we get all but major updates.

Or we also fix the minor version (`proposal = "~7.4"`) so that we don't automatically get all minor updates, but only fix releases.

But to be completely reproducible, the best idea is probably to always have a fixed proposal version for a fixed NuRadioMC tag/release.

Reply:
I would fix it for reproducibility (just to be on the safe side) to `proposal = "7.4.2"`

Other than that I think we can merge and then add additional changes on the NuRadio interface side in a separate PR